tendrl/ceph_integration/manager/user_request.py

```diff
@@ -237,6 +237,43 @@ def _submit(self, commands=None):
         return ceph.rados_commands(self.fsid, self._cluster_name, commands)
 
 
+class ECProfileRequest(UserRequest):
+    """A user request whose remote operations consist of librados mon commands
+
+    """
+
+    def __init__(self, headline, fsid, cluster_name, commands):
+        self._commands = commands
+        super(ECProfileRequest, self).__init__(headline, fsid, cluster_name)
+
+    def _submit(self, commands=None):
+        if commands is None:
+            commands = self._commands
+
+        LOG.debug("%s._submit: %s/%s" %
+                  (self.__class__.__name__, self._cluster_name, commands))
+        return ceph.ceph_command(self._cluster_name, commands)
+
+
+class RbdRequest(UserRequest):
```

Comment:
At the end if you see even the rados request would end up doing CLI only, so do the rbd request.